src/test/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ExceptionHandlerTest.kt

```diff
@@ -13,4 +17,15 @@ class ExceptionHandlerTest {
     assertThat(ex.code).isEqualTo("DUPLICATE")
     assertThat(ex.userMessage).contains("at 9:11 am on 8 December 2021")
   }
+
+  @Test
```

Comment:
Happy enough to keep this test, but I am questioning what its actually testing. I dont think its testing creating the exception as the test name suggests - I think its testing the internals of the exception class itself ?
Given the test class is ExceptionHandlerTest, and this test does nothing with the ExceptionHandler itself, I wonder if its in the wrong place?

Reply:
As discussed, the underlying problem here is that the error codes are all hanging off the bottom of the ExceptionHandler... but they have become complex enough to warrant their own source file now. So yes I will move the error codes and rename these tests in my next PR.